pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3770,6 +3770,21 @@ public boolean isDispatcherPauseOnAckStatePersistentEnabled() {
         return b == null ? false : b.booleanValue();
     }
 
+    @Override
+    public void updateDispatchPauseOnAckStatePersistentEnabled() {
+        super.updateDispatchPauseOnAckStatePersistentEnabled();
+        // Trigger new read if subscriptions has been paused before.
+        if (!brokerService.getPulsar().getConfiguration().isDispatcherPauseOnAckStatePersistentEnabled()) {
```

Comment:
```suggestion
        if (!topicPolicies.getDispatcherPauseOnAckStatePersistentEnabled().get())) {
```

Because the broker level configuration changes might not apply new policy to the topic, we have namespace, topic level policies.

Reply:
Fixed